.github/workflows/bleeding-edge.yml

```diff
@@ -0,0 +1,47 @@
+name: Bleeding Edge dependencies
+
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+  pull_request:
+  schedule:
+    - cron:  '0 0 * * 5'
+
+env:
+  INSTALL_EDM_VERSION: 3.2.3
+
+jobs:
+
+  # Test against EDM packages
+  test-with-edm:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache EDM packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache
+          key: ${{ runner.os }}-${{ matrix.toolkit }}-${{ hashFiles('etstool.py') }}
+      - name: Set up EDM
+        uses: enthought/setup-edm-action@v1
+        with:
+          edm-version: ${{ env.INSTALL_EDM_VERSION }}
+      - name: Install click to the default EDM environment
+        run: edm install -y wheel click coverage
+      - name: Install test environment
+        run: edm run -- python etstool.py install
+      - name: Remove dependencies obtanied with edm
+        run: |
+            edm plumbing remove-package --environment bootstrap --force traits
+            edm plumbing remove-package --environment bootstrap --force scipy
+      - name: Re-install dependencies using pip
+        run: |
+            edm run -e bootstrap -- python -m pip install --force-reinstall "git+http://github.com/enthought/traits.git#egg=traits"
+            edm run -e bootstrap -- python -m pip install --force-reinstall scipy
```

Comment:
I did not add `--no-deps` based on a comment I saw in envisage etstool:
```
# Without the --no-dependencies flag such that new dependencies on
# master are brought in.
```
Also I also used `--force-reinstall`.

Effectively I saw this PR https://github.com/enthought/chaco/pull/544 and followed.  This may not be necessary though. AFAIK `traits` has no required dependencies anyway, but `scipy` does